src/twister.h

```diff
@@ -8,8 +8,12 @@
 
 #define LIBTORRENT_PORT_OFFSET 1000
 
-#define USERPOST_FLAG_RT 0x01
-#define USERPOST_FLAG_DM 0x02
+#define USERPOST_FLAG_RT    0x01
+#define USERPOST_FLAG_DM    0x02
+#define USERPOST_FLAG_FAV   0x04
+#define USERPOST_FLAG_P_FAV 0x0C
```

Comment:
I just found a bit funny to have a bitfield flag with two bits set at once (P_FAV is 0x04 | 0x08 = 0x0C), but ok.
The semantic is more like FAV + PRIV, but i understand that PRIV alone (0x08) doesn't make much sense...